atomics/T1105/T1105.yaml

```diff
@@ -335,3 +335,48 @@ atomic_tests:
       del #{local_path} >nul 2>&1
       del %temp%\MpCmdRun.log >nul 2>&1
     name: command_prompt
+- name: whois file download
+  description: |
+    Download a remote file using the whois utility
+
+  supported_platforms:
+  - linux
+  - macos
+
+  input_arguments:
+    remote_host:
+      description: Remote hostname or IP address
+      type: String
+      default: localhost
+    remote_port:
+      description: Remote port to connect to
+      type: Integer
+      default: 8443
+    output_file:
+      description: Path of file to save output to
+      type: Path
+      default: /tmp/T1105.whois.out
+    query:
+      description: Query to send to remote server
+      type: String
+      default: "Hello from Atomic Red Team test T1105"
+    timeout:
+      description: Timeout period before ending process (seconds)
+      type: Integer
+      default: 1
+
+  dependencies:
+  - description: |
```

Comment:
I don't think a tick (`-`) should be before description?

Reply:
Looks like it's there in the test template: https://github.com/redcanaryco/atomic-red-team/blob/master/atomic_red_team/atomic_test_template.yaml#L23